gammapy/modeling/models/spatial.py

```diff
@@ -859,10 +864,11 @@ class DiskSpatialModel(SpatialModel):
         :math:`a`: length of the major semiaxis, in angular units.
         Default is 1 deg.
     e : `float`
-        Eccentricity of the ellipse (:math:`0< e< 1`).
+        Eccentricity of the ellipse (:math:`0<= e<= 1`).
         Default is 0.
     phi : `~astropy.coordinates.Angle`
-        Rotation angle :math:`\phi`: of the major semiaxis.
+        Rotation angle :math:`\phi`: of the major semiaxis (:math:`-20<=phi<=200`).
+        phi is wrapped between [0, 180] with the larger range enabled to enable fits for phi close to 0
```

Comment:
"with the larger range enabled to enable fits for phi close to 0"  : I think this is an implementation detail.

Reply:
removed